### PR TITLE
[B3.SQL] Fixed too long events giving a mysql error

### PR DESCRIPTION
**Pull request by Thomaxius**
CS:GO event Begin_Bomb_Defuse_With_Kit has 26 characters, whereas one can insert only a 25-character event into xlr_actionstats. This gives a mysql error.

### CODE REVIEW

File: b3/sql/sqlite/b3-update-1.10.0.sql [hunk 1]
```diff
@@ -4,6 +4,9 @@
 -- Add a table field for the xlr_playerstats table
 ALTER TABLE `xlr_playerstats` ADD `id_token` VARCHAR(10) NOT NULL DEFAULT '';
 
+-- Change xlr_actionstats so that one can supply an event of 26 characters
+ALTER TABLE  `xlr_actionstats` CHANGE  `name`  `name` VARCHAR( 26 )
```
Comment (thomasleveil) [inline b3/sql/sqlite/b3-update-1.10.0.sql]: This won't work as SQLite does not support altering a column. But also [SQLite does not either enforce length on TEXT type columns](http://www.sqlite.org/faq.html#q9), so there is no need for an update sql file if it aims to just change a column length.

**Comment by thomasleveil** [comment]
Thank you for contributing to the project :smile: 
Could you please revert back _b3/sql/sqlite/b3-update-1.10.0.sql_?

Also may I suggest to create your pull requests from different branches so the changes you add later on on one branch won't affect previous pull requests that aren't yet merged?

**Comment by thomasleveil** [comment]
oh and I notice this pull request hold a commit (11d1fea) which is a duplicate of 712f8949. The later one is already merged in our `release-1.10` branch. 

You end up with that situation because you pushes 712f8949 to your github `release-1.10` branch, now your `release-1.10` branch differs from ours and initiating pull requests from it would carry all commit that you have and that we don't have. That's another reason from initiating pull request from new dedicated branches. Here's the workflow to follow if you want to contribute a change:

1. on your home computer, create a new branch from the b3/release-1.10 branch and name it after the kind of change you intent to do (`fix-xxx`, `new_insurgency_event`, ...)
2. make some commits
3. test
4. push that branch to your github repo to a branch with the same name (and not `release-1.10`)
5. create the pull request from that branch

Let me know if you need guidance with that workflow ;)

**Comment by Thomaxius** [comment]
Sorry, I never used sqlite so I assumed it is the same.

And yeah, I'm new to this thing but I guess I got it right now. Should just probably stay away and leave this to the professionals..

**Comment by thomasleveil** [comment]
don't be sorry, I mysql learn mostly by making mistakes, the more I do the more I learn. Please keep contributing by either reporting bug, suggesting new feature and if you can propose code change through pull requests ;)

**Comment by danielepantaleone** [comment]
@Thomaxius don't worry. The first time I messed up with GIT it took me a week to understand what a pull request was, so you are already on a better path :smiley:

**Pull request closed without merging**